ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -3,12 +3,17 @@
 #
```

Comment:
I think this file is not best place for HPOPT tests, since OTE SDK has no HPOPT dependency. You can place your test somewhere in https://github.com/openvinotoolkit/training_extensions/tree/develop/tests/ote_cli

Reply:
As you suggested, HPO test is moved to ote_cli tests. Thank you!